apps/web/screens/LifeEvent/LifeEvent.tsx

```diff
@@ -184,7 +216,7 @@ LifeEvent.getInitialProps = async ({ apolloClient, locale, query }) => {
     throw new CustomNextError(404, 'Life Event not found')
   }
 
-  return { lifeEvent, namespace }
+  return { lifeEvent, namespace, locale: locale as Locale }
```

Comment:
Curious, seeing the locale define as Locale in the interface for the props, why is this happening?

Reply:
I noticed that most breadcrumbs in the code actually aren't pointing to english pages.
So I added the locale to the screen props and since the linkResolver expects a locale parameter of type Locale I typed it in the return here